src/FirmwarePlugin/APM/ArduCopterFirmwarePlugin.cc

```diff
@@ -64,7 +63,6 @@ ArduCopterFirmwarePlugin::ArduCopterFirmwarePlugin(void)
         APMCopterMode(APMCopterMode::CIRCLE,        true),
         APMCopterMode(APMCopterMode::LAND,          true),
         APMCopterMode(APMCopterMode::DRIFT,         true),
-        APMCopterMode(APMCopterMode::SPORT,         true),
```

Comment:
I believe that you should check the FW version and see if this mode is still valid.